### ISSUE TITLE
Extra space in blockquotes in Live Preview

### ISSUE THREAD

**Issue by Jklein64** [primary-theme/obsidian#116]
When editing a blockquote with the Live Preview setting enabled, the theme inserts extra space between lines.  Here's an example:

This is the original text.
<img width="719" alt="Screen Shot 2022-04-09 at 10 02 24 AM" src="https://user-images.githubusercontent.com/43863752/162577564-e64fbabf-6e66-4e83-bef8-2df379817b72.png">

This is me editing same text in a blockquote in the Live Preview.  Note the extra space between each of the paragraph breaks.
<img width="714" alt="Screen Shot 2022-04-09 at 10 03 07 AM" src="https://user-images.githubusercontent.com/43863752/162577588-b3fefa01-fdb2-4504-90ef-abcbb3e60d16.png">

This image shows the border boxes and content boxes of each of the quote elements.  It's clear that the padding is the cause of the spacing issue.  Note that there's an empty quote because two consecutive newlines separate paragraphs in quotes in markdown.
<img width="563" alt="Screen Shot 2022-04-09 at 10 12 29 AM" src="https://user-images.githubusercontent.com/43863752/162577929-62cacebe-9aa9-472e-94c7-bf20352f6b76.png">

Despite this, the reader view handles the rendering fine.
<img width="729" alt="Screen Shot 2022-04-09 at 10 04 22 AM" src="https://user-images.githubusercontent.com/43863752/162577629-d91e1a17-8253-4274-8aa5-a4a64da30cb9.png">

I spent some time trying to modify the CSS to fix this, but it's actually quite difficult.  When in the live preview, each line within the quote is a `<div>` with class `HyperMD-quote` (among others) *on the same level as the rest of the page's elements*.  In order to add spacing at the top and bottom of the quote, the theme currently adds padding to *each* of the quote elements, which affects the inside ones as well.

Ideally, what would happen in the Live Preview mode is the first element of each section of consecutive `.HyperMD-quote` elements would have padding on the top, and the last element of each consecutive chunk would have padding on the bottom.  I've written a CSS snippet to remove the top/bottom padding from all of the quote elements and then only add top padding to the first of each consecutive block, but I can't figure out how to do it for the last quote element, making the quote look off-balance.

```css
/* remove top/bottom padding for interior quote elements */
.HyperMD-quote {
    padding: 0 var(--scale-2-6) !important;
}

/* add top padding for the first quote element */
:not(.HyperMD-quote) + .HyperMD-quote {
    padding: var(--scale-2-6) var(--scale-2-6) 0 var(--scale-2-6) !important;
}
```

This fixes the spacing issue, but doesn't allow for the bottom-most quote element to be padded, which looks weird.
<img width="711" alt="Screen Shot 2022-04-09 at 10 19 28 AM" src="https://user-images.githubusercontent.com/43863752/162578181-557db4b2-9f8e-41d8-afe0-d1d5bdc95966.png">

Any ideas?

**Comment by huyz** [comment]
I get the same issue.  It's not pretty

**Comment by huyz** [comment]
Oh here's a workaround: add `> [!QUOTE]` before the blockquote.
When the cursor is inside the blockquote, the extra lines will still show.  But at least when the cursor is out, the rendering will look correct.

**Comment by vitamickey** [comment]
Having the same issue as y'all. Adding `[!QUOTE]` turns it into a callout, so the text on the first line becomes the title of the callout. It's an alright workaround, but would be nice to just fix it with another CSS workaround, to maintain the blockquote environment.

**Comment by ceciliamay** [comment]
i've since changed the designed for blockquotes to be more default to ensure consistency across viewing modes. this is available via the Primary 2.7.0 rebuild/refresh update!

![Image](https://github.com/primary-theme/obsidian/assets/90643719/746840e7-4011-4c16-af26-5a533b2be0e3)
